passport/package.json

```diff
@@ -27,6 +27,7 @@
   },
   "homepage": "https://github.com/reshufflehq/reshuffle#readme",
   "dependencies": {
+    "@reshuffle/auth": "0.1.0",
```

Comment:
[Technicality]: I think this should be "0.0.0", for rush tracking to work?

Reply:
I'll fix